aws/table_aws_ec2_spot_price.go

```diff
@@ -0,0 +1,125 @@
+package aws
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsEc2SpotPrice(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ec2_spot_price",
+		Description: "AWS EC2 Spot Price History",
+		List: &plugin.ListConfig{
+			Hydrate: listEc2SpotPrice,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: isNotFoundErrorV2([]string{"InvalidParameterValue"}),
```

Comment:
```suggestion
				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidParameterValue"}),
```

@aminvielledebatAtBedrock This was updated in a recent push, can you please update the function name?